docs/source/quick-start.mdx

```diff
@@ -25,20 +33,25 @@ To download a specific version of the file, use the `revision` parameter to spec
 
 ```py
 >>> from huggingface_hub import hf_hub_download
->>> hf_hub_download(repo_id="google/pegasus-xsum", filename="config.json", revision="4d33b01d79672f27f001f6abade33f22d993b151")
+>>> hf_hub_download(
+...     repo_id="google/pegasus-xsum", 
+...     filename="config.json", 
+...     revision="4d33b01d79672f27f001f6abade33f22d993b151"
+... )
 ```
 
-Finally, you can choose where to cache a file with the `cache_dir` parameter:
-
-```py
->>> from huggingface_hub import hf_hub_download
->>> hf_hub_download(repo_id="google/pegasus-xsum", filename="config.json", cache_dir="/my/cool/cache")
-```
+For more details and options, see the API reference for [`hf_hub_download`].
 
 ## Create a repository
 
 To create and share files to the Hub, you need to have a Hugging Face account. [Create an account]((https://huggingface.co/join)) if you don't already have one, and then sign in to find your [User Access Token](https://huggingface.co/docs/hub/security#user-access-tokens) in your Settings. The User Access Token is used to authenticate your identity to the Hub.
```

Comment:
[Create an account]((https://huggingface.co/join)) there are two parenthesis which redirects to (https://huggingface.co/join)